libpod/in_memory_state.go

```diff
@@ -933,6 +1058,44 @@ func (s *InMemoryState) removeCtrFromDependsMap(ctrID, dependsID string) {
 	}
 }
 
+// Add a container to the dependency mappings for the volume
+func (s *InMemoryState) addCtrToVolDependsMap(depCtrID, volName string) {
+	if volName != "" {
+		arr, ok := s.volumeDepends[volName]
+		if !ok {
+			// Do not have a mapping for that volume yet
+			s.volumeDepends[volName] = []string{depCtrID}
+		} else {
+			// Have a mapping for the volume
+			arr = append(arr, depCtrID)
+			s.volumeDepends[volName] = arr
+		}
+	}
+}
+
+// Remove a container from the dependency mappings for the volume
+func (s *InMemoryState) removeCtrFromVolDependsMap(depCtrID, volName string) {
+	if volName != "" {
+		arr, ok := s.volumeDepends[volName]
+		if !ok {
+			// Internal state seems inconsistent
+			// But the dependency is definitely gone
+			// So just return
+			return
```

Comment:
Probably sould be good to have at least a logrus.debug() here before the return.

Reply:
This is just for testing purposes I believe. Not sure if the logrus would really help.